general/general_ontology.ttl

```diff
@@ -0,0 +1,18 @@
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix gax-core: <https://w3id.org/gaia-x/core#> .
+@prefix general: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/general/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix gx: <https://registry.lab.gaia-x.eu/development/api/trusted-shape-registry/v1/shapes/jsonld/trustframework#> .
+
+general: a owl:Ontology ;
+    rdfs:label "ontology definition for General"@en ;
+    dcterms:contributor "Mirco Nierenz (TG)" ;
+    owl:versionInfo "0.1" .
+
+general:General a owl:Class ;
+    rdfs:label "class definition for General" ;
+    rdfs:comment "General attributes for all assets"@en ;
+    rdfs:subClassOf gx:DataResource .
```

Comment:
Muss general eine SubClass von DataResource sein? Reicht es nicht, wenn HD-Map, etc. das hat. Ein Angebot hat ja nie nur eine General Shape oder?

Reply:
Okay for now